src/app/features/common-components/registration/registration.component.html

```diff
@@ -165,6 +165,11 @@ <h4 class="data team-name"> {{ signUpForm.get("teamName").value }} </h4>
             <p class="category">Email</p>
             <p class="data"> {{ signUpForm.get("schoolEmailAddress").value }} </p>
           </div>
+          <div class="info-group">
```

Comment:
Is this where the tech stack of a participant is saved?

Reply:
By saved, do you mean where it's stored or where it's submitted? 

When the form is submitted, I believe the sign up service makes a request to the API/backend of the application to post the data from the sign up form. This specific portion of the page is where the user is able to review what they have inputted into the form so far before submitting the form and completing their registration. 

With the tech stack, when the form is submitted what should be sent to the backend is the names of the languages the participant knows from the checkboxes in the form of an array of strings and that's under the form control techStack. Any additional languages/frameworks will be stored in the otherLang form control and that will be just a string. I'm not sure if any of the information I provided answered your questions, so please let me know.

Reply:
okay, it is sent in a form of an array of strings. Awesome, that answers my question!